frontend/public/components/utils/headings.tsx

```diff
@@ -14,21 +14,29 @@ import {
   referenceForModel,
 } from '../../module/k8s';
 import { ResourceItemDeleting } from '../overview/project-overview';
+import { TechPreviewBadge } from '@console/shared';
+import './_breadcrumbs.scss';
 
 export const BreadCrumbs: React.SFC<BreadCrumbsProps> = ({breadcrumbs}) => (
-  <ol className="breadcrumb">
-    { breadcrumbs.map((crumb, i, {length}) => {
-      const isLast = i === length - 1;
-
-      return <li key={i} className={classNames({'active': isLast})}>
-        {isLast ? (
-          crumb.name
-        ) : (
-          <Link className="breadcrumb-link" to={crumb.path} data-test-id={`breadcrumb-link-${i}`}>{crumb.name}</Link>
-        )}
-      </li>;
-    }) }
-  </ol>);
+  <div className="co-breadcrumbs__row">
+    <div className="co-breadcrumbs__col">
+      <ol className="breadcrumb">
+        { breadcrumbs.map((crumb, i, {length}) => {
+          const isLast = i === length - 1;
+          return <li key={i} className={classNames({'active': isLast})}>
+          {isLast ? (
+            crumb.name
+          ) : (
+            <Link className="breadcrumb-link" to={crumb.path} data-test-id={`breadcrumb-link-${i}`}>{crumb.name}</Link>
+          )}
+          </li>;
+        })}
+      </ol>
+    </div>
+    <div className="co-breadcrumbs--tech-preview-badge">
+        <TechPreviewBadge />
```

Comment:
You are applying the `TechPreviewBadge` to all `Breadcrumbs` which is wrong.
This should be conditional and probably better controlled outside of the `Breadcrumbs` component and in the heading itself.

Reply:
Done!

Reply:
This isn't a good layout. Flow layouts will always be more acceptable than absolute positioned elements and less prone to breakages.
![image](https://user-images.githubusercontent.com/14068621/63622630-4518e780-c5c5-11e9-9e78-cd74633ca1e4.png)

You may be better off moving the addition of the tech preview badge from `Breadcrumbs` to `PageHeading` and applying a flex layout there with align-items center.

Reply:
I've added the badge to the `PageHeading` using `Split` layout.